cheevos-new/cheevos.c

```diff
@@ -380,11 +382,18 @@ static int rcheevos_parse(const char* json)
       lboard->info = rcheevos_locals.patchdata.lboards + j;
       res = rc_lboard_size(lboard->info->mem);
 
+      CHEEVOS_LOG(RCHEEVOS_TAG "rc_lboard_size() = %d\n", res);
+
       if (res < 0)
       {
-         CHEEVOS_ERR(RCHEEVOS_TAG "Error in leaderboard mem %s: %s",
-            lboard->info->mem, rcheevos_rc_error(res));
-         goto error;
+         snprintf(buffer, sizeof(buffer), "Error in leaderboard %d \"%s\"", lboard->info->id, lboard->info->title);
```

Comment:
An average user probably doesn't care either way, but I would have worded this to indicate that it was a Parse Error, probably even including the error code.

Reply:
oh yeah, I'll add the `rcheevos_rc_error(res)` call here. Even if average users don't know what the message means, it helps when they care enough to report the problem.
thanks for the input :+1: